minimint-bridge/Cargo.toml

```diff
@@ -11,22 +11,22 @@ crate-type = ["staticlib"]
 anyhow = "1"
 flutter_rust_bridge = "1"
 
-mint-client = { git = "https://github.com/justinmoon/minimint", branch = "plugin" }
-minimint-api = { git = "https://github.com/justinmoon/minimint", branch = "plugin" }
-minimint = { git = "https://github.com/justinmoon/minimint", branch = "plugin" }
+mint-client = { git = "https://github.com/fedimint/minimint" }
+minimint-api = { git = "https://github.com/fedimint/minimint" }
+minimint = { git = "https://github.com/fedimint/minimint" }
```

Comment:
Let's pin to a commit. For example, see the `confirm_invoice` comment below.

Reply:
cargo doesn't like that, I tried in 66b95ed
```
Updating git repository `https://github.com/fedimint/minimint`
error: failed to resolve patches for `https://github.com/fedimint/minimint`

Caused by:
  patch for `minimint` in `https://github.com/fedimint/minimint` points to the same source, but patches must point to different sources
```

Reply:
I think not specifying `rev` is fine. 

> Once a git dependency has been added, Cargo will lock that dependency to the latest commit at the time. New commits will not be pulled down automatically once the lock is in place. However, they can be pulled down manually with cargo update.

(from cargo book)

It has a nice output too. 
![image](https://user-images.githubusercontent.com/49202620/178655541-9697398e-02e9-4bab-bccc-52ed47fa2c46.png)